package.json

```diff
@@ -25,13 +25,12 @@
   "author": "REI Software Engineering",
   "license": "MIT",
   "peerDependencies": {
-    "@rei/cdr-tokens": ">=9.0.0",
-    "@rei/cedar": ">=9.0.0"
```

Comment:
This doesn't need Cedar as a peerDependency and with the changes to NPM installing them by default I'm going to save us an extra maintenance headache.